account_due_list/account_move_line.py

```diff
@@ -45,6 +45,66 @@ class AccountMoveLine(models.Model):
                                         compute='_get_invoice',
                                         string='Invoice', store=True)
 
+    maturity_residual = fields.Float(
+        compute='_maturity_residual', string="Residual Amount", store=True,
+        help="The residual amount on a receivable or payable of a journal "
+             "entry expressed in the company currency.")
+
+    @api.multi
+    @api.depends('date_maturity')
```

Comment:
The fields I listed above are  used by ```amount_residual```.
That is, for example, if you don't put ```reconcile_partial_id.line_partial_ids``` and if a move.line is added to ```reconcile_partial_id.line_partial_ids``` , ```maturity_residual``` is not recomputed